app.py

```diff
@@ -58,5 +60,18 @@ def manual_scrobbling():
     task.start_scrobbling()
     return redirect("/")
 
+@app.route("/test")
+def test_scrobble():
```

Comment:
Ah, this was just me testing the code, and I needed a name for my function. The whole route will be removed when I'm done. Sorry for the confusion.